spec/tasks/whitehall_importer_spec.rb

```diff
@@ -5,6 +5,13 @@
 
   let(:import_data) { whitehall_export_with_one_edition }
 
+  before do
+    binary_image = File.open(File.join(fixtures_path, "files", "960x640.jpg"), "rb").read
+    stub_request(:get, "https://assets.publishing.service.gov.uk/government/uploads/valid-image.jpg").to_return(status: 200, body: binary_image)
+    stub_request(:get, "https://assets.publishing.service.gov.uk/government/uploads/another-valid-image.jpg").to_return(status: 200, body: binary_image)
+    stub_request(:get, "https://assets.publishing.service.gov.uk/government/uploads/foo/valid-image.jpg").to_return(status: 200, body: binary_image)
+  end
```

Comment:
This block seems to be quite impactful to performance running before all the tests. I saw it increase from 1 second to 17 seconds execution time. Maybe we can just do this for an image test?

Reply:
On it 💪

Reply:
Just marked this as unresolved as I'd expect we need to move this code into the images context.